src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinInetProxyHelper.cs

```diff
@@ -79,6 +82,10 @@ public bool ManualSettingsOnly
 
         public string ProxyBypass { get; set; }
 
+        public bool RecentAutoDetectionFailure =>
+            _autoDetectionFailed &&
+            Environment.TickCount - _lastTimeAutoDetectionFailed <= _recentAutoDetectionInterval;
```

Comment:
Shouldn't this be `Environment.TickCount - _lastTimeAutoDetectionFailed > _recentAutoDetectionInterval`?

If lastError == Interop.WinHttp.ERROR_WINHTTP_AUTODETECTION_FAILED (`_autoDetectionFailed` = true), we will cache the information and call `GetProxyForUrl` 2 minutes later, right?

(Probably here causes the slight perf issue)